docs/6.0. Upgrade Path.md

```diff
@@ -8,47 +8,47 @@ API versioning is specified in the [APIs](2.0.%20APIs.md) documentation, with pr
 
 ## Requirements for Nodes (Node APIs)
 
-Node API implementations MUST support at least one API version, and MAY support more than one at a time. Note however that a Node MUST only perform interactions with a Registration API at a single version. Nodes implementing multiple API versions MAY provide a user-configurable choice for which API version to register and/or query using. Otherwise it is expected that the Node will select a Registration API which supports the highest API version which the Node also supports. The API version SHOULD be selected before considering the Registration API's configured \'priority\'.
+Node API implementations MUST support at least one API version, and MAY support more than one at a time. Note however that a Node MUST only perform interactions with a Registration API at a single version. Nodes implementing multiple API versions MAY provide a user-configurable choice for which API version to register and/or query using. Otherwise it is expected that the Node will select a Registration API which supports the highest API version which the Node also supports. The API version SHOULD be selected before considering the Registration API's configured `priority`.
 
 Registrations MUST NOT proceed if the Node API version implemented does not exactly match the API version used by the Registration API.
 
-Where a Node implements version v1.2 or below, it MUST browse for both the \'\_nmos-register.\_tcp\' DNS-SD service type, and the legacy \'\_nmos-registration.\_tcp\' DNS-SD service type in order to retrieve the full list of available Registration APIs. De-duplication SHOULD be performed against this returned list.
+Where a Node implements version v1.2 or below, it MUST browse for both the `_nmos-register._tcp` DNS-SD service type, and the legacy `_nmos-registration._tcp` DNS-SD service type in order to retrieve the full list of available Registration APIs. De-duplication SHOULD be performed against this returned list.
```

Comment:
(Another instance of "version vX.Y".)

Reply:
Let's deal with the issues around saying "version vX.Y" or "version X.Y" and the hash preceding MAJOR and MINOR or not in the parent PR.